### ISSUE TITLE
ESPNet2 for aishell at stage 9

### ISSUE THREAD

**Issue by im73** [espnet/espnet#3228]
**Describe the bug**
I use ESPnet2 for aishell, but with my own datasets. Some configuration are as follows:
- lm_config conf/train_lm_transformer.yaml   
- asr_config conf/train_asr_transformer.yaml(**the default is train_asr_conformer.yaml,Because I encountered some problems in use, so I modified it**)
- inference_config conf/decode_asr_rnn.yaml 
The problems I encountered are as follows:
1. when I'm training asr at stage 10, the log information in the log file show that
<img width="1152" alt="image" src="https://user-images.githubusercontent.com/36858700/119076940-06c48c80-ba26-11eb-86af-b1d45eb80660.png">
the "There are no improvements in this epoch" appeared in every epoch, and stop at epoch 20 (the max epoch used in my configuration is 200)but from the loss_ctc, loss_att and loss, we could find that the loss decreased 
<img width="507" alt="image" src="https://user-images.githubusercontent.com/36858700/119078371-a3882980-ba28-11eb-8525-bd857f8a532b.png">
<img width="491" alt="image" src="https://user-images.githubusercontent.com/36858700/119078377-a84cdd80-ba28-11eb-9c83-6a25ffa03dff.png">
<img width="479" alt="image" src="https://user-images.githubusercontent.com/36858700/119078386-aedb5500-ba28-11eb-9feb-4ba573b6fcfd.png">

 but the car and wer always are 1.0:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/36858700/119080219-42625500-ba2c-11eb-8077-89af3495a51f.png">

Is there something wrong with my usage? 
2. If the audio time is too short in the training data(All the audio in the data set is only about 3s), will it lead to any problems ? or  will there be measures to deal with the situation of short time audio in the training progress？because judging from the current results, it has performed poorly,I wonder if it's my training problem or my training data.
3. In the stage 10(training ars), will it load the trained lm_model in the stage 7(LM training) to train the whole asr model？
**Basic environments:**
 - OS information: Linux 5.4.0-72-generic #80-Ubuntu SMP Mon Apr 12 17:35:00 UTC 2021 x86_64
- python version: `3.6.13 |Anaconda, Inc.| (default, Feb 23 2021, 21:15:04)  [GCC 7.3.0]`
- espnet version: `espnet 0.9.9`
- pytorch version: `pytorch 1.4.0`
 - Git hash [e.g. b88e89fc7246fed4c2842b55baba884fe1b4ecc2]
- Git hash: `55160fc20c48686808cbdf208919e668e3b1451a`
  - Commit date: `Mon Apr 26 18:21:56 2021 -0400`
